tests/apt/test_protocol.py

```diff
@@ -290,6 +293,49 @@ def test_AptMessage_MGMSG_MOT_REQ_POSCOUNTER_to_bytes() -> None:
     assert msg.to_bytes() == b"\x11\x04\x01\x00\x50\x01"
 
 
+def test_AptMessage_MGMSG_MOT_GET_STATUSUPDATE_from_bytes() -> None:
+    msg = AptMessage_MGMSG_MOT_GET_STATUSUPDATE.from_bytes(
+        bytes.fromhex("8104 0e00 81 22 01000100 000000000000 07000000")
```

Comment:
The formatting here doesn't make it easy to understand which parts correspond to which values.
```suggestion
        bytes.fromhex("8104 0e00 81 22 0100 01000000 00000000 07000000")
```

Was this pulled out of the logs? If not, do we have any actual values that we can test with?

Reply:
The `enc_count` part is taken from the documentation example, the rest are the same as the test for `USTATUSUPDATE` message. 
"Enc Counts: 00, 00, 00, 00: Only used with encoded stages"